README.md

```diff
@@ -1,27 +1,25 @@
-# bpmn-visualization
+<h1 align="center">BPMN Visualization</h1> <br>
+<p align="center">
+   <img title="BPMN Visualization" src="https://github.com/process-analytics/bpmn-visualization-js/blob/master/config/logo.png" width="812" height="170">
```

Comment:
:question: is it possible to use an image not pointing to the master branch to keep the README and the file in sync and not having maintenance issue?

Reply:
done in 2042c49